ubuntu/BUILD

```diff
@@ -1,19 +1,55 @@
 package(default_visibility = ["//visibility:public"])
 
 load("//dockerfile_build:dockerfile_build.bzl", "dockerfile_build")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
 
 UBUNTU_VERSIONS = [
     "16_0_4",
 ]
 
 [dockerfile_build(
-    name = "ubuntu_%s" % version,
-    base = "//ubuntu/reproducible:ubuntu_%s_vanilla" % version,
+    name = "ubuntu_%s_dockerbuild" % version,
+    base = ":ubuntu_%s_vanilla" % version,
     dockerfile = ":Dockerfile.ubuntu",
 ) for version in UBUNTU_VERSIONS]
 
 [dockerfile_build(
-    name = "ubuntu_%s_build" % version,
-    base = "//ubuntu/reproducible:ubuntu_%s" % version,
+    name = "ubuntu_%s_build_dockerbuild" % version,
+    base = ":ubuntu_%s" % version,
     dockerfile = ":Dockerfile.ubuntu_build",
 ) for version in UBUNTU_VERSIONS]
+
+UBUNTU_ENV = {
+    "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+}
+
+# The overlay directory contains files that should be overlaid into the rootfs at /.
+pkg_tar(
+    name = "overlay",
+    srcs = glob(["overlay/**/*"]),
+    package_dir = "/",
+    strip_prefix = "overlay/",
+)
+
+# 'ubuntu_%s_vanilla' is docker image derived from a snapshot ubuntu_%s rootfs
```

Comment:
Can you leave some comments indicating why we have both flavors left in here? I assume its for comparison/testing later?

Reply:
These targets are copied from https://github.com/GoogleCloudPlatform/base-images-docker/blob/master/ubuntu/reproducible/BUILD. I will definitely add some comments.